src/test/java/seedu/address/logic/commands/SortCustomerCommandTest.java

```diff
@@ -0,0 +1,278 @@
+package seedu.address.logic.commands;
```

Comment:
Hi @chetwinlow you missed out on several classes like `SortCustomerCommandParser` and `SortByCustomer`. Can't remember if thats all off the top of my head. Please review and add all test cases.